### ISSUE TITLE
Bit depth of the original video

### ISSUE THREAD

**Issue by gzsun1416** [chxy95/HDRTVNet#12]
Hi,
 I have a question.  As you say，the dataset is extracted from the video of HDR10 standard (10-bit, Rec.2020, PQ) and stored in 16 bit PNG .  However, when I read an HDR image with MATLAB, I found that the image contains more than 60000 pixel intensities instead of 1024. This means that this is a real 16 bit image.   Is the  original video really just 10 bits?

**Comment by chxy95** [comment]
@gzsun1416 Yes, you can use ffmpeg or Mediainfo to check the bit depth of the video. FFmpeg can not extract images in 10-bit format.
